src/agent/src/device.rs

```diff
@@ -56,9 +56,50 @@ struct DevIndexEntry {
 #[derive(Debug)]
 struct DevIndex(HashMap<String, DevIndexEntry>);
 
+fn filename_is_pci_addr(f: &std::ffi::OsStr) -> bool {
```

Comment:
Could you add some unit tests for this please?

Reply:
Good idea, adding that for the next spin.